doc/src/tools/kani.md

```diff
@@ -46,29 +47,28 @@ Verification failed for - harness
 Complete - 0 successfully verified harnesses, 1 failures, 1 total.
 ```
 
+For a more detailed tutorial, you can refer to the [tutorial section of the Kani book](https://model-checking.github.io/kani/kani-tutorial.html).
 
 ## Using Kani to verify the Rust Standard Library
 
-To aid the Rust Standard Library verification effort, Kani provides a sub-command out of the box to help you get started.
+Here is a short tutorial to get started with using Kani to verify some proofs that you write for the standard library.
 
-### Step 1
+### Step 1 - Add some proofs to your copy of the model-checking std
 
-Modify your local copy of the Rust Standard Library by writing proofs for the functions/methods that you want to verify.
+Create a local copy of the model-checking fork of the Rust Standard Library. The fork comes with Kani initiated, so all you'll need to do is to call Kani's
+building-block APIs (such as
+`assert`, `assume`, `proof` and [function-contracts](https://github.com/model-checking/kani/blob/main/rfc/src/rfcs/0009-function-contracts.md) such as `proof_for_contract` and `fake_function`) directly.
 
-For example, insert this short blob into your copy of the library. This blob imports the building-block APIs such as
-`assert`, `assume`, `proof` and [function-contracts](https://github.com/model-checking/kani/blob/main/rfc/src/rfcs/0009-function-contracts.md) such as `proof_for_contract` and `fake_function`.
+For example, insert this module into an existing file like `library/core/src/lib.rs` in your copy of the library.
```

Comment:
Shouldn't be `lib.rs` -- they mainly just declare their modules in there, that's not where any real logic will actually go. I'd recommend choosing a file where we already have harnesses, like `core::ptr:unique` or something. (That also has the added benefit of letting people learn from our example).
If you'd prefer not to choose a file where we already have harnesses (so that they do it from scratch), that's fine by me, but I would pick something in core that isn't `lib.rs`.

Reply:
I've fixed this, lmk if the newer wording works.